app/templates/main/search-transferring-body.html

```diff
@@ -255,46 +255,41 @@ <h2 class="govuk-heading-m govuk-heading-m--search">
                                 <h3 class="govuk-heading-s govuk-heading-s--search-term">Search terms applied</h3>
                                 <div class="ayr-filter-tags">
                                     {% for i in range(search_terms | length) %}
-                                        <div class="search-term">
-                                            {% set terms_to_keep = [] %}
-                                            {% for j in range(search_terms | length) %}
-                                                {% if j != i %}
-                                                    {% set term = search_terms[j] %}
-                                                    {% if term.strip() %}
-                                                        {% if ' ' in term and not (term.startswith('"') and term.endswith('"')) %}
-                                                            {% set term = '"' + term + '"' %}
-                                                        {% endif %}
-                                                        {% set _ = terms_to_keep.append(term) %}
+                                        {% set terms_to_keep = [] %}
+                                        {% for j in range(search_terms | length) %}
```

Comment:
I don 't quite understand this nested for loop logic but I know it's not new.

Since we are here can we rename `i` to `search_term_index ` and `j` to `remaining_term_index` or some other duo of descriptive names?